tinyrpc/protocols/jsonrpc.py

```diff
@@ -508,6 +508,7 @@ class JSONRPCProtocol(RPCBatchProtocol):
     def __init__(self, *args, **kwargs) -> None:
         super(JSONRPCProtocol, self).__init__(*args, **kwargs)
         self._id_counter = 0
+        self._incomplete_requests = []
```

Comment:
May I suggest a more descriptive name like 'awaiting_replies' or 'pending_requests'?
Good thing to create an array for it!

Reply:
Updated it to be '_pending_replies'.